test/suites/migration.sh

```diff
@@ -34,12 +29,8 @@ test_migration() {
     [ -d "${LXD_DIR}/snapshots/nonlive2/snap0/rootfs/bin" ]
   fi
 
-  # FIXME: make this backend agnostic
-  if [ "${LXD_BACKEND}" = "dir" ]; then
-    # since we didn't snapshot above, we can't do this here
-    lxc copy l1:nonlive2/snap0 l2:nonlive3
-    [ -d "${LXD2_DIR}/containers/nonlive3/rootfs/bin" ]
-  fi
+  lxc copy l1:nonlive2/snap0 l2:nonlive3
+  [ -d "${LXD2_DIR}/containers/nonlive3/rootfs/bin" ]
```

Comment:
That line will fail with lvm as rootfs isn't mounted unless the container is running.